source/PlayerInfo.cpp

```diff
@@ -2127,10 +2179,25 @@ void PlayerInfo::Visit(const System &system)
 
 
 
+bool PlayerInfo::OutfitIsKnown(const Outfit& outfit) const
```

Comment:
functions should be declared in the same order as they are declared in the header, please change this